### PR TITLE
Migrated automanaged namespaces in node-throughput and windows-test tests

### LINKED ISSUE #1696: Remove deprecated automanagedNamespaces
AutomanagedNamespaces has been deprecated but is still being used in `node-throughput` and `windows-tests` configs. These configs are used in sig-scalability periodic jobs and sig-windows tests.

### PR DESCRIPTION
**Pull request by rahulii**
**What type of PR is this?**
/kind bug
/kind failing-test

**What this PR does / why we need it**:
This PR removes deprecated automanagedNamespaces
**Which issue(s) this PR fixes**:

Part Of #1696 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by rahulii** [comment]
Hi @mm4tt @krzysied , in #1694, @lyzs90 has mentioned to clean-up the automanagedNamespaces references, i believe i also need to make changes to those files , any help would be awesome!

**Comment by rahulii** [comment]
I signed it

**Comment by MadhavJivrajani** [comment]
/remove-kind failing-test

Not sure if this is a failing test :), just deprecated

File: clusterloader2/testing/node-throughput/config.yaml [hunk 1]
```diff
@@ -7,9 +7,17 @@
...
+
+#variables
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+
+{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 1 $schedulerThroughputNamespaces}}
 
 name: node-throughput
-automanagedNamespaces: {{$POD_COUNT}}
+namespace:
+  number: {{AddInt $namespaces $schedulerThroughputNamespaces}}
```
Comment (marseel) [inline clusterloader2/testing/node-throughput/config.yaml:20]: It should be just "number: {{$POD_COUNT}}".
Other things you copied from load/config.yaml are not necessary ;)

File: clusterloader2/testing/windows-tests/config.yaml [hunk 2]
```diff
@@ -7,9 +7,17 @@
...
+#variables
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+
+{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 1 $schedulerThroughputNamespaces}}
+
 
 name: node-throughput
-automanagedNamespaces: {{$POD_COUNT}}
+namespace:
+  number: {{AddInt $namespaces $schedulerThroughputNamespaces}}
```
Comment (marseel) [inline clusterloader2/testing/windows-tests/config.yaml:20]: Same as above

**Comment by rahulii** [comment]
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/api/validation.go#L47

Should i also remove all these conditions where `automanagedNamespaces` is used?

**Comment by rahulii** [comment]
> We can do it in next step ;)
> /ok-to-test

Should I create a new issue for this ?

**Comment by marseel** [comment]
I think we can do it as a part of the same issue ;) You just need to remove "Fixes #..." so it doesn't close issue. 
/lgtm

**Comment by marseel** [comment]
Ah, also please squash your commits, thanks :)

**Comment by marseel** [comment]
Can you put more descriptive commit message?
Something like "Migrated automanaged namespaces in node-throughput and windows-test tests"?
Thanks again for contribution :)

**Comment by rahulii** [comment]
> Can you put more descriptive commit message?
> Something like "Migrated automanaged namespaces in node-throughput and windows-test tests"?
> Thanks again for contribution :)

Done!

Should i create new PR and remove all the places where automanaged namespace is used ?
@marseel

**Comment by marseel** [comment]
Yes, sounds good ;) 

/lgtm

**Pull request merged**